Microsoft.Toolkit.Uwp.UI.Controls/RotatorTile/RotatorTile.cs

```diff
@@ -49,6 +49,18 @@ public sealed class RotatorTile : Control
         private WeakEventListener<RotatorTile, object, NotifyCollectionChangedEventArgs> _inccWeakEventListener;
 
         /// <summary>
+        /// Identifies the <see cref="MinRandomDuration"/> property.
+        /// </summary>
+        public static readonly DependencyProperty MinRandomDurationProperty =
+            DependencyProperty.Register(nameof(MinRandomDuration), typeof(int), typeof(RotatorTile), new PropertyMetadata(default(int)));
+
+        /// <summary>
+        /// Identifies the <see cref="MinRandomDuration"/> property.
+        /// </summary>
+        public static readonly DependencyProperty MaxRandomDurationProperty =
```

Comment:
Having one property will limit developers imo. Lets say a dev wanted to have at least 15 seconds for every tile and up to 50 seconds. If we have one property you will eliminate that at least 15 seconds rule.

Default values won't break existing behavior since it's still being randomized if non of the properties are set.

We can have others' opinion on int to TimeSpan thing. I can rename the properties like 'InSeconds' or change the type to TimeSpan.

Waiting for a latest clearification before preparing the docs. pinging @deltakosh